workshops/lib/twilio-basic/signup.md

```diff
@@ -4,185 +4,82 @@ By the end of this tutorial, you will have a Twilio account SID and
-
-> ![](img/upgrade_account.png)
-
-You will see this screen:
-
-> ![](img/upgrade_your_account.png)
-
-### Credit Card Info
-
-Although Twilio is giving each member $100 worth of credit, you still
-need a credit card to signup for Twilio.
-
-For your convenience, we have provided a credit card to signup with below. You
-may use your own card as well (your card will not be charged if you turn off
-automatic recharging).
-
-| Field              | Value                               |
-|--------------------|-------------------------------------|
-| Credit Card Number | `4847356897256173`                  |
-| Expires            | `09/2021`                           |
-| CVV                | `917`                               |
-| Cardholder Name    | `Jonathan Leung`                    |
-| Street Address     | `200 Haverford Road, Wynnewood, PA` |
-| Zip Code           | `19096`                             |
-
-We enter this info below:
-
-> ![](img/enter_credit_card_info.gif)
-
-Leave the "Company/Project URL" and "Phone Number" fields blank
-
-> ![](img/leave_blank.png)
-
-### Entering The Promo Code
-
-Click "I have a promo code."
-
-> ![](img/i_have_a_promo_code.png)
-
-To get the promo code:
-
-  - Club members: Ask your club leader for the Twilio promo code.
-
-  - Club leaders: send an email to the email address
-    `jonathanleung1337@gmail.com` with your full name and school and ask for the
-    Twilio promo code. They will respond to you quickly.
+1. Open [https://www.twilio.com/signup](https://www.twilio.com/)
+2. Fill in your information
+  - Note that there are 3 questions like "What are you building?". Don't bother
+    answering them correctly. Just choose the first answer you see and move on.
+3. Click "Get Started" button to move to the next step
+4. Twilio then asks you to enter a phone number. Do that.
+5. Click "Text Me" and continue following remaining directions until you see
+   this screen:
+   > ![](img/finish_verification_screen.png)
 
-After obtaining the promo code:
+6. Then scroll all the way down:
+   > ![](img/scroll_down.gif)
 
-- Enter the promo code
-- Uncheck "Turn on automatic recharge"
+7. Click the "Upgrade your account" button
+   > ![](img/upgrade_account.png)
 
-> ![](img/enter_promo_code.gif)
+8. You will see this screen:
+   > ![](img/upgrade_your_account.png)
 
-After making sure that all the information you entered is correct, click the
-"Upgrade Account" button.
+9. Fill in the below information.
+    - Leave the "Company/Project URL" and "Phone Number" fields blank
+    - For the credit card information, although Twilio is giving each member
+      $100 worth of credit, you still need a credit card to signup for Twilio.
+      For your convenience, we have provided a credit card to signup with below.
 
-> ![](img/click_upgrade_account.png)
+      | Field              | Value                               |
+      |--------------------|-------------------------------------|
```

Comment:
Every table created by someone that's not you follows this format (as far
as I'm aware). I'll add this to the Markdown style guide.

On Tue, Feb 2, 2016 at 12:07 PM, Jonathan Leung <notifications@github.com>
wrote:

> In workshops/lib/twilio-basic/signup.md
> <https://github.com/hackclub/hackclub/pull/667#discussion_r51626676>:
>
> >
> > -> ![](img/click_upgrade_account.png)
> > +      | Field              | Value                               |
> > +      |--------------------|-------------------------------------|
>
> Therefore I don't think it makes sense to all of a sudden enforce this now
> especially when it's not in the style guide.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/hackclub/pull/667/files#r51626676>.
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>